lib/codec.js

```diff
@@ -0,0 +1,165 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {termCodecs} from './codecs/codecMap';
+import {Base64PadCodec} from './codecs/base64Pad';
+import {CborldError} from './error';
+import {CodecMapCodec} from './codecs/codecMap';
+import {UrlCodec} from './codecs/url';
+import {ContextCodec} from './codecs/context';
+import {SimpleTypeCodec} from './codecs/simpleType';
+import {XsdDateTimeCodec} from './codecs/xsdDateTime';
+
+/**
+ * Gets a list of all term codecs available to use when transforming term
+ * values.
+ *
+ * @returns {Array} - An array of all term codecs.
+ */
+export function getTermCodecs() {
+  const codecs = [];
+  termCodecs.forEach((value, key) => {
+    if(typeof key === 'string') {
+      codecs.push(key);
+    }
+  });
+
+  return codecs;
+}
+
+/**
+ * Gets a compression map for the given set of JSON-LD Context URLs.
+ *
+ * @param {object} [args] - The arguments to the function.
```

Comment:
We usually use "options" here. If there's no reason to deviate, I recommend we stick with it.

```suggestion
 * @param {object} options - The arguments to the function.
```

Reply:
Also, this isn't actually an optional param, right?